============================= test session starts =============================
platform win32 -- Python 3.6.5, pytest-3.5.1, py-1.5.3, pluggy-0.6.0 -- C:\Users\reasonold\Anaconda3\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\reasonold\codenation\data-science-0, inifile:
plugins: remotedata-0.2.1, openfiles-0.3.0, doctestplus-0.1.3, arraydiff-0.2
collecting ... collected 10 items

test_submit.py::test_q1 PASSED                                           [ 10%]
test_submit.py::test_q2 PASSED                                           [ 20%]
test_submit.py::test_q3 PASSED                                           [ 30%]
test_submit.py::test_q4 PASSED                                           [ 40%]
test_submit.py::test_q5 PASSED                                           [ 50%]
test_submit.py::test_q6 PASSED                                           [ 60%]
test_submit.py::test_q7 PASSED                                           [ 70%]
test_submit.py::test_q8 PASSED                                           [ 80%]
test_submit.py::test_q9 PASSED                                           [ 90%]
test_submit.py::test_q10 PASSED                                          [100%]

- generated xml file: C:\Users\reasonold\codenation\data-science-0\output.xml -
============================== warnings summary ===============================
test_submit.py::test_q7
  C:\Users\reasonold\codenation\data-science-0\main.py:191: FutureWarning: 'argmax' is deprecated, use 'idxmax' instead. The behavior of 'argmax'
  will be corrected to return the positional maximum in the future.
  Use 'series.values.argmax' to get the position of the maximum now.
    q7=black_friday['Product_Category_3'].value_counts().argmax()

-- Docs: http://doc.pytest.org/en/latest/warnings.html
==================== 10 passed, 1 warnings in 2.83 seconds ====================
